### PR TITLE
Update terminology

### PR DESCRIPTION
**Pull request by hkny**
Terminology has been updated. Ready to be reviewed.

### CODE REVIEW

File: spec/terminology.md [hunk 1]
```diff
@@ -1,12 +1,39 @@
...
+~ A verifier is an entity that validates identity information from a holder to grant access to goods and services.
+
+[[def: Holder]]
+~ A holder, also known as an identity holder, is the entity that is in possession of the credentials. In most use cases, the holder is also the identity subject. A holder can interact with an issuer to obtain anonymous credentials. It can also create derived information from anonymous credentials that can be presented to a verifier to gain access to goods and services.
+
+[[def: SCHEMA]]
+
+~ A SCHEMA object is a template defining the set of attributes (also known as names, claims) in an AnonCreds verifiable credential. A SCHEMA must be accessible to all verifiable credential participants, issuers, holders, and verifiers.
+
+[[def: CLAIM_DEF]]
```
Comment (swcurran) [inline spec/terminology.md:27]: I think "CRED_DEF" should be used, with "CLAIM_DEF" as a synonym. Both are in use in the code, but there has been a transition away from CLAIM_DEF and to the more accurate CRED_DEF.

We also have to decide on whether what this is. Is it the object on the ledger?  Is it the data structure that goes on the ledger?  What do we call the "internal cred def" that the issuer stores that holds the private keys corresponding to the public keys in the CRED_DEF?

File: spec/terminology.md [hunk 2]
```diff
@@ -1,12 +1,39 @@
...
+[[def: REV_REG_DEF]]
+~ A REV_REG_DEF object (short for “revocation registry definition”) contains information required for verifiers in order to enable them to verify whether a (revokable) verifiable credential has been revoked by the issuer since issuance. A REV_REG_DEF is only required for revokable AnonCreds and is written to the ledger by the owner (issuer) of the CLAIM_DEF immediately after the CLAIM_DEF is written. 
+
+[[def: REV_REG_ENTRY]]
+~ A REV_REG_ENTRY object (short for “revocation registry entry”) marks the current status of one or more revokable verifiable credentials (“revoked” or “not revoked”) in the ledger in a privacy preserving manner. A REV_REG_ENTRY can only be written by the owner of a REV_REG_DEF, who is also the owner of CLAIM_DEF. 
 
-::: todo
 Finalize glossary entries
 
 **Question**: Should the items that are AnonCreds data models be included in this?
```
Comment (swcurran) [inline spec/terminology.md:39]: My leaning is that the data models be inline in the Data Flow section, perhaps with a link from here to there. But I'm not certain that's the best idea.

File: spec/terminology.md [hunk 3]
```diff
@@ -1,12 +1,39 @@
...
+[[def: Verifier]]
+
+~ A verifier is an entity that validates identity information from a holder to grant access to goods and services.
+
+[[def: Holder]]
+~ A holder, also known as an identity holder, is the entity that is in possession of the credentials. In most use cases, the holder is also the identity subject. A holder can interact with an issuer to obtain anonymous credentials. It can also create derived information from anonymous credentials that can be presented to a verifier to gain access to goods and services.
+
+[[def: SCHEMA]]
+
+~ A SCHEMA object is a template defining the set of attributes (also known as names, claims) in an AnonCreds verifiable credential. A SCHEMA must be accessible to all verifiable credential participants, issuers, holders, and verifiers.
```
Comment (brentzundel) [inline spec/terminology.md:25]:
```suggestion
~ A SCHEMA object is a template defining the set of claims (also known as names, attributes) in an AnonCreds verifiable credential. A SCHEMA must be accessible to all verifiable credential participants, issuers, holders, and verifiers.
```

File: spec/terminology.md [hunk 4]
```diff
@@ -1,12 +1,39 @@
 ## Terminology
 
 These terms are defined by this specification:
+[[def: Subject]]
+~ A subject, also known as an identity subject, is the entity that a digital identity belongs to.
```
[the remark this answers is not in the document]
Comment (hkny) [reply spec/terminology.md:5]: sounds better, will put in the next commit

File: spec/terminology.md [hunk 5]
```diff
@@ -1,12 +1,39 @@
...
+
+[[def: Issuer]]
+~ An issuer is one of the three entities that interact with each other within the domain of digital identities. It can assert claims about a subject in the form of a tamper-proof credential whose origins are cryptographically verifiable. 
+
+[[def: Verifier]]
+
+~ A verifier is an entity that validates identity information from a holder to grant access to goods and services.
+
+[[def: Holder]]
+~ A holder, also known as an identity holder, is the entity that is in possession of the credentials. In most use cases, the holder is also the identity subject. A holder can interact with an issuer to obtain anonymous credentials. It can also create derived information from anonymous credentials that can be presented to a verifier to gain access to goods and services.
```
[the remark this answers is not in the document]
Comment (hkny) [reply spec/terminology.md]: will be changed in the next commit

File: spec/terminology.md (hunk 3 above)
Comment (hkny) [reply spec/terminology.md]: will be in the next commit

File: spec/terminology.md [hunk 6]
```diff
@@ -1,12 +1,39 @@
...
+
+[[def: SCHEMA]]
+
+~ A SCHEMA object is a template defining the set of attributes (also known as names, claims) in an AnonCreds verifiable credential. A SCHEMA must be accessible to all verifiable credential participants, issuers, holders, and verifiers.
+
+[[def: CLAIM_DEF]]
+
+~ A CLAIM_DEF object derived from a SCHEMA contains key information regarding the issuer. A SCHEMA is in a 1:n relation with CLAIM_DEF, meaning there can be many CLAIM_DEFs related to a SCHEMA while a CLAIM_DEF can only derive from one SCHEMA. A CLAIM_DEF must be accessible to all verifiable credential participants, issuers, holders, and verifiers.
```
[the remark this answers is not in the document]
Comment (hkny) [reply spec/terminology.md]: took the definition from the indy method. Contains the information you suggested

File: spec/terminology.md (hunk 1 above)
Comment (hkny) [reply spec/terminology.md]: Added / Changed accordingly. Please check if we are happy with the private_CRED_DEF description (and terminology)

**Pull request closed without merging**